vmdb/lib/workers/event_catcher_kubernetes.rb

```diff
@@ -5,7 +5,7 @@ def event_monitor_handle
     require 'kubernetes/events/kubernetes_event_monitor'
     @event_monitor_handle ||= KubernetesEventMonitor.new(
       @ems.api_endpoint,
-      @ems.api_version
+      EmsKubernetes.kubernetes_version
```

Comment:
Does it ever make sense that we would have different versions for different instances of EmsKubernetes?  I'm wondering if there should be an instance method that just delegates to the class method.  Then, in the future, you could potentially have that method become a column or something else and wouldn't have to change things all over.

On that note, is `.api_version` on the EMS used at all?  Maybe it should be?